Header header

string              type

std_msgs/Float64    curvature

float64[]           left_fit 

float64[]           right_fit 